Jenkins testing for adding comment function

FernandoAndrade-NOAA opened the pull request:
This is a test PR for adding comment functionality to Jenkins UPP RT Pipeline.

ankimball committed:
Adding comment function

ankimball committed:
Updating addresses to personal repo

ankimball commented:
Jenkins-ci ORTs passed

ankimball committed:
changing token back to epic cicd

epic-cicd-jenkins commented:
Jenkins-ci ORTs passed

ankimball committed:
testing token for build status

ankimball commented:
Jenkins-ci ORTs passed

FernandoAndrade-NOAA committed:
Update rt.sh 

Exit code 1 if results changed and text file for Jenkins to output changes.

ankimball committed:
updating comment body for failures

ankimball committed:
Merge branch 'feature/jenkins-comment' of https://github.com/ankimball/UPP into feature/jenkins-comment

FernandoAndrade-NOAA committed:
sample bl change for fail test

ankimball commented:
Jenkins-ci ORTs failed

ankimball committed:
Update Jenkinsfile echo messages for debug

ankimball commented:
Jenkins-ci ORTs failed

ankimball committed:
Updating location of txt file

ankimball commented:
Jenkins-ci ORTs failed

ankimball committed:
moving post section into Hera

ankimball commented:
Hera Jenkins-ci ORTs failed

ankimball committed:
printing txt file for debug

ankimball commented:
Hera Jenkins-ci ORTs failed

ankimball committed:
Update Jenkinsfile

ankimball committed:
fixing double quotes

ankimball committed:
changing read file function

ankimball committed:
changing comment body format

ankimball committed:
changing comment message format

ankimball committed:
escaping \ character

ankimball committed:
testing message part

ankimball committed:
change comment format

ankimball committed:
adding prints

ankimball committed:
trying vaired comment message tests

ankimball committed:
posting just original message

ankimball commented:
Hera Jenkins-ci ORTs failed

ankimball committed:
Update Jenkinsfile

ankimball committed:
Update Jenkinsfile escape newlines

ankimball commented:
Hera Jenkins-ci ORTs failedThere are changes in results for case fv3r_pe_test in PRSLEV10.tm00nThere are changes in results for case fv3r in PRSLEV10.tm00nThere are changes in results for case rtma_pe_test in PRSLEV00.tm00nThere are changes in results for case rtma in PRSLEV00.tm00nRefer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.n

ankimball committed:
Update Jenkinsfile

ankimball commented:
Hera Jenkins-ci ORTs failedThere are changes in results for case fv3r_pe_test in PRSLEV10.tm00 There are changes in results for case fv3r in PRSLEV10.tm00 There are changes in results for case rtma_pe_test in PRSLEV00.tm00 There are changes in results for case rtma in PRSLEV00.tm00 Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
update comment message

ankimball committed:
adding escape for newlines

ankimball committed:
fixing escape characters

ankimball commented:
Jenkins-ci HERA ORTs failed: There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
add newline before diff info

ankimball commented:
Jenkins-ci HERA ORTs failed: \nThere are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
fixing first newline

ankimball commented:
Jenkins-ci HERA ORTs failed: \nThere are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
fix first newline

ankimball commented:
Jenkins-ci HERA ORTs failed:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
code cleanup

ankimball commented:
Hera UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
move logic to function

ankimball committed:
add global variable

ankimball committed:
fix variable placements

ankimball committed:
fixing typo

ankimball committed:
add back missing line

ankimball commented:
Hera UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
Update access token

ankimball committed:
code cleanup

ankimball committed:
fixing token

ankimball committed:
fixing typo

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
add function to other platforms

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
test change naming convention

epic-cicd-jenkins commented:
none UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
changing machine name convention back

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
set up parallel stages

ankimball committed:
fix pass comment message

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED:
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED:\n

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED:\n

ankimball committed:
fix newlines

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED:\n

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
fixing formatting, adding extra newline

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
testing on jet only

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

ankimball committed:
testing on orion only

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
testing hercules only

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

ankimball committed:
run orion and hera

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
testing all

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
switching order

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
running just orion

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
add back hercules

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

ankimball committed:
switch Hercules and hera

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
full test again

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Orion UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
adding sleeps

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

ankimball committed:
edit sleeps

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

ankimball committed:
adding retries

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
add try catch for retry

ankimball committed:
remove extra bracket

ankimball committed:
add script step

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests PASSED

ankimball committed:
remove try catch block

ankimball committed:
Merge branch 'NOAA-EMC:develop' into feature/jenkins-comment

FernandoAndrade-NOAA committed:
Update Jenkinsfile add Hercules

Adds Hercules Jenkins steps, should be the same as Orion. rt.sh should be creating separate machine-specific test directories.

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in NATLEV10.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case rtma in NATLEV00.tm00
There are changes in results for case hrrr_pe_test in WRFTWO.GrbF04
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case fv3r in NATLEV10.tm00
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case nmmb in BGDAWP03.tm00.Grib2
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma_pe_test in NATLEV00.tm00
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case hrrr in WRFTWO.GrbF04
There are changes in results for case nmmb_pe_test in BGDAWP03.tm00.Grib2
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-HERCULES for details on differences in results for each case.

ankimball committed:
try different method of retry

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

epic-cicd-jenkins commented:
Hera UPP Regression Tests PASSED

ankimball committed:
run orion alone

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

ankimball committed:
retry orion

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
orion and jet

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

ankimball committed:
Orion, jet, and Hercules

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

ankimball committed:
move orion to the end

ankimball committed:
fix parentheses

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

epic-cicd-jenkins commented:
Jet UPP Regression Tests PASSED

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in NATLEV10.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case rtma in NATLEV00.tm00
There are changes in results for case hrrr_pe_test in WRFTWO.GrbF04
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case fv3r in NATLEV10.tm00
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case nmmb in BGDAWP03.tm00.Grib2
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma_pe_test in NATLEV00.tm00
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case hrrr in WRFTWO.GrbF04
There are changes in results for case nmmb_pe_test in BGDAWP03.tm00.Grib2
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-HERCULES for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

ankimball committed:
Merge branch 'NOAA-EMC:develop' into feature/jenkins-comment

ankimball committed:
move orion location

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in NATLEV10.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case rtma in NATLEV00.tm00
There are changes in results for case hrrr_pe_test in WRFTWO.GrbF04
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case fv3r in NATLEV10.tm00
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case nmmb in BGDAWP03.tm00.Grib2
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma_pe_test in NATLEV00.tm00
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case hrrr in WRFTWO.GrbF04
There are changes in results for case nmmb_pe_test in BGDAWP03.tm00.Grib2
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-HERCULES for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in NATLEV10.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case rtma in NATLEV00.tm00
There are changes in results for case hrrr_pe_test in WRFTWO.GrbF04
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case fv3r in NATLEV10.tm00
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case nmmb in BGDAWP03.tm00.Grib2
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma_pe_test in NATLEV00.tm00
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case hrrr in WRFTWO.GrbF04
There are changes in results for case nmmb_pe_test in BGDAWP03.tm00.Grib2
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-HERCULES for details on differences in results for each case.

ankimball committed:
testing non parallel method

ankimball committed:
skip hera due to maintenance

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in NATLEV10.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case rtma in NATLEV00.tm00
There are changes in results for case hrrr_pe_test in WRFTWO.GrbF04
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case fv3r in NATLEV10.tm00
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case nmmb in BGDAWP03.tm00.Grib2
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma_pe_test in NATLEV00.tm00
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case hrrr in WRFTWO.GrbF04
There are changes in results for case nmmb_pe_test in BGDAWP03.tm00.Grib2
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-HERCULES for details on differences in results for each case.

ankimball committed:
Re-add Hera and clean up

epic-cicd-jenkins commented:
Orion UPP Regression Tests FAILED

There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-ORION for details on differences in results for each case.

epic-cicd-jenkins commented:
Hera UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma in PRSLEV00.tm00
Refer to .diff files in rundir: /scratch2/NAGAPE/epic/role.epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1/ci/rundir/upp-HERA for details on differences in results for each case.

epic-cicd-jenkins commented:
Hercules UPP Regression Tests FAILED

There are changes in results for case gfs_pe_test in gfs.t00z.master.grb2f006
There are changes in results for case fv3r_pe_test in NATLEV10.tm00
There are changes in results for case fv3r_pe_test in PRSLEV10.tm00
There are changes in results for case rtma in PRSLEV00.tm00
There are changes in results for case rtma in NATLEV00.tm00
There are changes in results for case hrrr_pe_test in WRFTWO.GrbF04
There are changes in results for case gfs in gfs.t00z.master.grb2f006
There are changes in results for case fv3r in PRSLEV10.tm00
There are changes in results for case fv3r in NATLEV10.tm00
There are changes in results for case rap_pe_test in WRFPRS.GrbF16
There are changes in results for case rap_pe_test in WRFNAT.GrbF16
There are changes in results for case nmmb in BGDAWP03.tm00.Grib2
There are changes in results for case rtma_pe_test in PRSLEV00.tm00
There are changes in results for case rtma_pe_test in NATLEV00.tm00
There are changes in results for case rap in WRFPRS.GrbF16
There are changes in results for case rap in WRFNAT.GrbF16
There are changes in results for case hrrr in WRFTWO.GrbF04
There are changes in results for case nmmb_pe_test in BGDAWP03.tm00.Grib2
Refer to .diff files in rundir: /work/noaa/epic/role-epic/jenkins/workspace/UPP_UPP-Pipeline_PR-1_2/ci/rundir/upp-HERCULES for details on differences in results for each case.

ankimball committed:
Updating comment destination to NOAA-EMC repo

FernandoAndrade-NOAA committed:
revert dummy bl change

WenMeng-NOAA committed:
Remove 3 hybrid level variables from RRFS products. (#836)

* Remove 3 hybrid level variables from RRFS products.

* Add fix for ci failture

FernandoAndrade-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/jenkins-comment
